test/runner/run.ts

```diff
@@ -70,7 +71,7 @@ export async function generateNewOutput(list: any) {
   });
 }
 
-export async function writeNewOutput(list: any, update: boolean) {
+export async function writeNewOutput(list: any, update: string | boolean) {
   let updated = 0;
```

Comment:
the func param here is `update` and it's linked to `opts.update`. If false it will fail and throw. 
Issue is that this will always be falsy on normal testing `npm test`. So the runner will always fail running tests